src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTestBase.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class ProcessTestBase : IDisposable
+    {
+        protected const int WaitInMS = 100 * 1000;
+        protected const string CoreRunName = "corerun";
+        protected const string TestExeName = "System.Diagnostics.Process.TestConsoleApp.exe";
+        protected const int SuccessExitCode = 100;
+
+        protected Process _process;
```

Comment:
Now every derived test class is going to have its own test process.  Is that what we want?